### PR TITLE
Explain Performance Counter User Group Installer

### PR DESCRIPTION
**Pull request by boblangley**
WIP: Wording, possibly explain _why_ this is necessary

### CODE REVIEW

**Comment by boblangley** [comment]
Triggering Desk case: https://nservicebus.desk.com/agent/case/16349

**Comment by boblangley** [comment]
@SimonCropp Would you care to review?

File: nservicebus/operations/performance-counters.md [hunk 1]
```diff
@@ -111,6 +111,9 @@ Enabled by default and will only write to the counter if it exists.
 
 The NServiceBus Performance counters can be installed using the [NServiceBus PowerShell tools](management-using-powershell.md).
 
+## Perfomance Monitor Users local security group
+
+When using NServiceBus.Host.exe or BusConfiguration.EnableInstallers() the service account will be automatically added to the local Performance Monitor Users group if executed with elevated priviledges.
```
Comment (SimonCropp) [inline nservicebus/operations/performance-counters.md]: `BusConfiguration.EnableInstallers()` is not cross version friendly. link here instead http://docs.particular.net/nservicebus/operations/installers

Comment (SimonCropp) [inline nservicebus/operations/performance-counters.md]: > NServiceBus.Host.exe

Link here instead http://docs.particular.net/nservicebus/hosting/nservicebus-host/

File: nservicebus/operations/performance-counters.md [hunk 2]
```diff
@@ -111,6 +111,9 @@ Enabled by default and will only write to the counter if it exists.
 
 The NServiceBus Performance counters can be installed using the [NServiceBus PowerShell tools](management-using-powershell.md).
 
+## Perfomance Monitor Users local security group
+
+When [running installers](http://docs.particular.net/nservicebus/operations/installers) the service account will be automatically added to the local Performance Monitor Users group if executed with elevated privileges.
```
Comment (SimonCropp) [inline nservicebus/operations/performance-counters.md]: links cant be fully qualified https://github.com/Particular/docs.particular.net#linking

File: nservicebus/operations/performance-counters.md [hunk 3]
```diff
@@ -111,6 +111,9 @@ Enabled by default and will only write to the counter if it exists.
 
 The NServiceBus Performance counters can be installed using the [NServiceBus PowerShell tools](management-using-powershell.md).
 
+## Perfomance Monitor Users local security group
+
+When [running installers](installers) the service account will be automatically added to the local Performance Monitor Users group if executed with elevated privileges.
```
Comment (SimonCropp) [inline nservicebus/operations/performance-counters.md]: `installers` will be a broken link

**Pull request merged**